content/en/project/news/_index.md

```diff
@@ -5,24 +5,24 @@ type: docs
 aliases:
  - /hugo/news/
 ---
-<iframe width="560" height="315" src="https://www.youtube.com/embed/N1MobfEaoWY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-<p>
+### 2023 Annual Report (in preparation)
 
-The **2022 Medley Interlisp [Annual Report](/medley/project/status/2022medleyannualreport)** provides a detailed report on 2022 accomplishments.
+Sneek Peek: We are working on our [2023 Medley Interlisp Annual Report](https://docs.google.com/document/d/1yL1WeYvgu5JstWyjD3fWl1mDZv8kIr_U3_1onSWvUko/edit?usp=sharing). Still a work in progress -- don't distribute until we announce it is ready.
 
-## Recent Updates
+### 3/2023
 
-## 12/2022
+Bay Area Lisp talk:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/N1MobfEaoWY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+## 12/2022
 * [2022 Annual Report](/medley/project/status/2022medleyannualreport)
 
 ## 9/2022
-
```

Comment:
Markdown lists are supposed to be surrounded by blank lines.  

https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md032.md